src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -84,21 +91,32 @@ export const LeafyGreenSelect: React.FC<WidgetProps> = ({
           aria-labelledby={ariaLabelledBy}
           label={ariaLabelledBy ? undefined : label}
           value={value}
-          onChange={(v) => onChange(v)}
+          onChange={(v) => onChange(v === "" ? null : v)}
           placeholder={placeholder}
           data-cy={dataCy}
         >
-          {enumOptions.map((o) => (
-            <Option key={o.value} value={o.value}>
-              {o.label}
-            </Option>
-          ))}
+          {enumOptions.map((o) => {
+            // Handle deselect value without errors
+            if (o.value === null) {
+              return;
+            }
+            return (
+              <Option key={o.value} value={o.value}>
+                {o.label}
+              </Option>
+            );
+          })}
         </Select>
+        {hasError && <Error>Selection is required.</Error>}
       </MaxWidthContainer>
     </ElementWrapper>
   );
 };
 
+const Error = styled(Description)`
+  color: ${uiColors.red.base};
```

Comment:
nit. We usually destructure `uiColors`  at the top of the file. and only reference the destructured value in styles. 
```suggestion
  color: ${red.base};
```